django_oemof/views.py

```diff
@@ -66,7 +67,7 @@ def post(request):
         parameters = hooks.apply_hooks(
             hook_type=hooks.HookType.SETUP, scenario=scenario, data=parameters, request=request
         )
-        task = simulation.simulate_scenario.delay(scenario, parameters)
+        task = current_app.tasks["django_oemof.simulation.simulate_scenario"].delay(scenario, parameters)
```

Comment:
Why do you make use of `current_app`?
Is this a good practice? Did not read about this yet...